5-app-infra/modules/env_base/main.tf

```diff
@@ -88,3 +95,116 @@ module "compute_instance" {
   instance_template     = module.instance_template.self_link
   resource_manager_tags = local.resource_manager_tags
 }
+
```

Comment:
Todo esse bloco que foi adicioando do confidential tem que ser movido para um modulo novo ou tem que ter um comndicional nos recusos para somemte serem criado se for a chamada do .

do modo que o codigo esta sendo feito  ser chamado 3 veze seguidas: module "gce_instance", module "peering_gce_instance" e para o seu modulo novo

Reply:
melhor criara um novo modulo chamado `confidencial_space`, vai ter muita coisa nele